packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/exampleInsightPanel.js

```diff
@@ -58,7 +63,18 @@ export default class ExampleInsightPanel extends LightningElement {
       'objecttype',
       ['FeedItem']
     );
+    const isThread = headless.ResultTemplatesHelpers.fieldMustMatch(
+      'source',
+      ['iNaturalistTaxons']
```

Comment:
What is that? Can't we find something else that makes sense that isn't talking about Taxons? lol

Reply:
you can ask @mmitiche more about taxons lol.  I just took the same isThread from the exampleSearch X)

Reply:
In the org `Search UI Samples` used by default with Quantic, that's only source of folded results we could find. That's why we keep using it in out examples.